From 4b22ba780ca53ff9c9c85e310066b354c4c20516 Mon Sep 17 00:00:00 2001
From: Bharat Kumar Gogada <bharat.kumar.gogada@xilinx.com>
Date: Mon, 13 Jan 2020 15:26:30 +0530
Subject: [PATCH 0834/1851] PCI: XDMA PL PCIe: Fix number of MSI interrupts
 supported

commit 140b29a4abad3bebd5e8b4f6e719697301628b12 from
https://github.com/Xilinx/linux-xlnx.git

PCI XDMA IP supports only 32 MSI interrupts, fixing with
correct number.

Signed-off-by: Bharat Kumar Gogada <bharat.kumar.gogada@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: not-upstreamable
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/pci/controller/pcie-xdma-pl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/controller/pcie-xdma-pl.c b/drivers/pci/controller/pcie-xdma-pl.c
index a32df034a9ac..1c99efdb187f 100644
--- a/drivers/pci/controller/pcie-xdma-pl.c
+++ b/drivers/pci/controller/pcie-xdma-pl.c
@@ -90,7 +90,7 @@
 #define ECAM_DEV_NUM_SHIFT		12
 
 /* Number of MSI IRQs */
-#define XILINX_NUM_MSI_IRQS		128
+#define XILINX_NUM_MSI_IRQS		32
 #define INTX_NUM                        4
 
 struct xilinx_msi {
-- 
2.31.1

